merlin/systems/triton/utils.py

```diff
@@ -1,20 +1,23 @@
 import contextlib
 import os
 import signal
+import socket
 import subprocess
 import time
 from distutils.spawn import find_executable
 
 import tritonclient
 import tritonclient.grpc as grpcclient
 
-import merlin.systems.triton as triton
+from merlin.systems import triton
 
 TRITON_SERVER_PATH = find_executable("tritonserver")
 
 
 @contextlib.contextmanager
-def run_triton_server(modelpath):
+def run_triton_server(
+    modelpath, grpc_host="localhost", grpc_port=None, backend_config="tensorflow,version=2"
+):
     """This function starts up a Triton server instance and returns a client to it.
 
     Parameters
```

Comment:
Could you update the docstring to describe the new parameters and maybe a brief description of how the port selection works now?